### PR TITLE
fix(gc64) fix some `.sxx` to adjust to `LuaJIT-2.1-20210510`

### PR DESCRIPTION
**Pull request by ADD-SP**

### CODE REVIEW

**Review by hutchic — APPROVED** [review]
looks sensible enough

File: samples/lj-gc-objs.sxx
Comment (dndx) [inline samples/lj-gc-objs.sxx:98]: Note: relavant commit https://github.com/openresty/luajit2/commit/ff34b48ddd6f2b3bdd26d6088662a214ba6b0288

File: samples/lj-gc-objs.sxx
Comment (dndx) [inline samples/lj-gc-objs.sxx]: Need to double check the iterator code to make sure it is traversing the hash table correctly. 

https://github.com/openresty/luajit2/commit/ff34b48ddd6f2b3bdd26d6088662a214ba6b0288?diff=split#diff-dd7c01bcb65d865239c0db1aa0950fd29fd8bf84f1a31449aec73e7dae470501

File: samples/lj-gc.sxx
Comment (dndx) [inline samples/lj-gc.sxx]:
```suggestion
            if (retries > @RETRIES_THRESHOLD) {
```

Comment (dndx) [reply samples/lj-gc.sxx]: Also, need to double check and make sure OpenResty was compiled with debug on, otherwise optimization may cause strange runtime behavior.

**Comment by kikito** [comment]
Note for reviewers: hide whitespace-only changes for easier review.

In addition to the `DO NOT MERGE IT`, message I have converted the PR to a Draft PR, which is not mergeable. Anyone should be able to change it back to mergeable once it is ready to be merged.

File: samples/lj-gc-objs.sxx
Comment (fffonion) [reply samples/lj-gc-objs.sxx]: i saw this one and the following changes are being removed in the latest commit, is it because the original change is incorrect?

**Pull request closed without merging**
